packages/rspack/src/utils/versions.ts

```diff
@@ -7,4 +7,4 @@ export const lessLoaderVersion = '~11.1.3';
 export const sassLoaderVersion = '^16.0.4';
 export const sassEmbeddedVersion = '^1.83.4';
 
-export const reactRefreshVersion = '~0.14.0';
+export const reactRefreshVersion = '~0.15.0';
```

Comment:
Oopsie, sorry, just realised I unintentionally changed that, I'm reverting this line